### PR TITLE
Nikola/remove marketplace from sample main

### LINKED ISSUE #1139: Remove unnecessary Marketplaces calls in SampleMain
Now that we have **MarketplaceSampleScene**, we can remove this samples from "SampleMain":
Screenshot 2024-09-09 at 5 22 51 PM

### PR DESCRIPTION
**Pull request by kantagara**
Removed marketplace from sample main
Removed EVM class -> moved some things to Web3Unity if people _really_ need to do some things like contract send and contract call (will not encourage them to do so :D )
All tests are passing on my end.
Made a build with WebGL.

closes #1139

### CODE REVIEW

**Review by juans-chainsafe — COMMENTED** [review]
Looks like Unit tests are failing

**Comment by kantagara** [comment]
Ah yes 🫠  not because of my changes tho, they're failing on dev as well. Wtf.

**Review by juans-chainsafe — CHANGES_REQUESTED** [review]
Looks good! Marketplace samples are not anymore in SampleMain and is building as expected.

But unit tests are failing

**Comment by juans-chainsafe** [comment]
> Ah yes 🫠 not because of my changes tho, they're failing on dev as well. Wtf.

oh right, but they were working in Rob PRs, I saw the green check there, but now they are failing

**Review by juans-chainsafe — APPROVED** [review]

**Comment by rob1997** [comment]
> > Ah yes 🫠 not because of my changes tho, they're failing on dev as well. Wtf.
> 
> oh right, but they were working in Rob PR, I saw the green check there, but now they are failing

green check was probably for CLA

**Review by sneakzttv — APPROVED** [review]

File: Packages/io.chainsafe.web3-unity.web3auth/Runtime/Web3AuthTransactionExecutor.cs [hunk 1]
```diff
@@ -48,7 +49,7 @@ public async void TransactionApproved(string transactionId)
         var response = await base.SendTransaction(transaction.request);
             
         transaction.response.SetResult(response);
-
+        Debug.Log(response.Hash);
```
Comment (rob1997) [inline Packages/io.chainsafe.web3-unity.web3auth/Runtime/Web3AuthTransactionExecutor.cs:52]: do we need this log?

File: Packages/io.chainsafe.web3-unity/Editor/ABICSharpConverter.cs [hunk 2]
```diff
@@ -337,7 +362,7 @@ private string ParseEventSubscription()
             sb.Append(eventSubscription);
             sb.Append("\n");
         }
-
+        Debug.LogError(sb);
```
Comment (rob1997) [inline Packages/io.chainsafe.web3-unity/Editor/ABICSharpConverter.cs:365]: is this log needed?

File: Packages/io.chainsafe.web3-unity/Runtime/Resources/ABIContractClassTemplate.txt [hunk 3]
```diff
@@ -49,27 +50,63 @@ namespace ChainSafe.Gaming.Evm.Contracts.Custom
...
+            
            
-            _webSocketClient ??= new StreamingWebSocketClient(WebSocketUrl);
-            await _webSocketClient.StartAsync();
-            Subscribed = true;
+            try
+            {
+                if(Application.platform == RuntimePlatform.WebGLPlayer)
+                {
+                   Debug.LogWarning("WebGL Platform is currently not supporting event subscription");
```
Comment (rob1997) [inline Packages/io.chainsafe.web3-unity/Runtime/Resources/ABIContractClassTemplate.txt:87]: even with async utils doesn't work?

**Review by rob1997 — APPROVED** [review]

File: Packages/io.chainsafe.web3-unity/Runtime/Resources/ABIContractClassTemplate.txt (hunk 3 above)
Comment (kantagara) [reply Packages/io.chainsafe.web3-unity/Runtime/Resources/ABIContractClassTemplate.txt:87]: No because this uses native http websockets from .net 😢

**Pull request merged**